source/includes/language-compatibility-table-csharp.rst

```diff
@@ -37,7 +37,7 @@
      - .NET 4.7 [#2.14-note]_
      - .NET 4.6
 
-   * - 2.13 to 2.29
+   * - 2.13 to 2.30
```

Comment:
Do we need to update the table just above this one too? Currently it only goes to 2.29

Reply:
good catch!